config/options.go

```diff
@@ -225,6 +225,11 @@ type Options struct {
 	// DataBrokerURL is the routable destination of the databroker service's gRPC endpiont.
 	DataBrokerURLString string   `mapstructure:"databroker_service_url" yaml:"databroker_service_url,omitempty"`
 	DataBrokerURL       *url.URL `yaml:",omitempty"`
+	// DataBrokerBackendStorageType is the storage backend type that databroker will use.
+	// Supported type: memory, redis
+	DataBrokerBackendStorageType string `mapstructure:"databroker_backend_storage_type" yaml:"databroker_backend_storage_type,omitempty"`
```

Comment:
what about DataBrokerStorageType and DataBrokerConnectionString or... something like that? DSN is a bit ambitious to me.

Reply:
ok